app/assets/javascript/application.js

```diff
@@ -9,6 +10,12 @@ jQuery(function($) {
   });
 
   ////
+  // Add "select all"/"clear all" buttons to each select2 dropdown menu
+  GOVUK.DropdownSelectAll({
+    $el: $("select.select2")
```

Comment:
Having a chat with @tijmenb we decided to start by applying it to every dropdown by default, and only adding in a toggle if there are specific ones that we don't want to show this on.